tests/test_program.py

```diff
@@ -14,18 +14,6 @@ def test_raise() -> None:
         assert f"{e}" == "('clvm raise', '86666f6f626172')"
 
 
-def test_raise_program() -> None:
```

Comment:
I would expect this test to still work and be relevant. we're not looking at the return value even

Reply:
re-added now with the new run func